### PR TITLE
Set citus.node_conninfo sslmode to --ssl-mode.

### PR DESCRIPTION
**Pull request by DimCitus**

### CODE REVIEW

File: src/bin/pg_autoctl/primary_standby.c [hunk 1]
```diff
@@ -65,6 +65,7 @@ GUC postgres_default_settings[] = {
 GUC citus_default_settings[] = {
 	DEFAULT_GUC_SETTINGS_FOR_PG_AUTO_FAILOVER,
 	{ "shared_preload_libraries", "'citus'" },
+	{ "citus.node_coinninfo", "'sslmode=prefer'" },
```
Comment (JelteF) [inline src/bin/pg_autoctl/primary_standby.c]:
```suggestion
	{ "citus.node_conninfo", "'sslmode=prefer'" },
```

File: src/bin/pg_autoctl/pgctl.c [hunk 2]
```diff
@@ -500,6 +500,11 @@ prepare_guc_settings_from_pgsetup(const char *configFilePath,
 								  setting->name, pgSetup->ssl.serverKey);
 			}
 		}
+		else if (streq(setting->name, "citus.node_coinninfo"))
```
Comment (JelteF) [inline src/bin/pg_autoctl/pgctl.c]:
```suggestion
		else if (streq(setting->name, "citus.node_conninfo"))
```

File: src/bin/pg_autoctl/pgctl.c [hunk 3]
```diff
@@ -500,6 +500,11 @@ prepare_guc_settings_from_pgsetup(const char *configFilePath,
 								  setting->name, pgSetup->ssl.serverKey);
 			}
 		}
+		else if (streq(setting->name, "citus.node_conninfo"))
+		{
+			appendPQExpBuffer(config, "%s = '%s'\n",
```
Comment (JelteF) [inline src/bin/pg_autoctl/pgctl.c]: there should be all the SSL related connection option in here, i.e. `sslrootcert` and `sslcrl` (maybe more)

Comment (DimCitus) [reply src/bin/pg_autoctl/pgctl.c]: Done now, good point.

**Pull request merged**